src/lib/TranscriptEditor/index.module.css

```diff
@@ -1,32 +1,69 @@
 @value color-subt-green, color-darkest-grey, color-labs-red from './colours.module.css';
 
+::selection {
+  background: color-subt-green;
+}
+
+::moz-selection {
+  background: color-subt-green;
+}
+
+.help {
```

Comment:
Moving this hasn't quite worked. Styling is a bit weird + space is too narrow. I remember this being fiddly..

before:
<img width="294" alt="screen shot 2019-02-21 at 14 19 21" src="https://user-images.githubusercontent.com/9170575/53175371-b7dbe280-35e3-11e9-9b48-bfd45a87ecc1.png">

after:
<img width="226" alt="screen shot 2019-02-21 at 14 21 25" src="https://user-images.githubusercontent.com/9170575/53175517-038e8c00-35e4-11e9-907a-a20eae788f1f.png">

Reply:
yeah, very fiddely but there is room to fix this 

eg 
```
.help {
  width: 48%;
  text-align: center;
```

<img width="976" alt="screen shot 2019-02-28 at 19 34 53" src="https://user-images.githubusercontent.com/4661975/53593189-f51a1480-3b8f-11e9-97e1-32b94742d8df.png">